res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -1138,6 +1175,36 @@ TraktorS4MK2.callbackLoopMove = function(field) {
   }
 }
 
+TraktorS4MK2.sendLoopSizeMessage = function(group, beats) {
+  var LoopSizeIndicators = {
+      "deck2" : [0x2B, 0x3A],
+      "deck1" : [0x1B, 0x2A],
+  }
+  start_signal = LoopSizeIndicators[group]
+
+  var sizesArray = {
```

Comment:
I don't suppose it's possible to mark this const somehow?

Reply:
QtScript 4.8 doesn't support such modern JavaScript keywords.